### PR TITLE
Updated ordering of tabs and instructions

### PR DESCRIPTION
**Pull request by dhairyya**
Spoofing tab order was incorrect
Cmdlet to update entries missed the correct heading or had incomplete information

### CODE REVIEW

File: microsoft-365/security/office-365-security/modify-remove-entries-tenant-allow-block.md [hunk 1]
```diff
@@ -65,19 +67,19 @@ You can use the Microsoft 365 Defender portal or PowerShell to modify and remove
...
-
+ 
 3. Select the entry that you want to remove, and then click ![Delete icon.](../../media/m365-cc-sc-delete-icon.png) **Delete**.
 
 4. In the warning dialog that appears, click **Delete**.
 
 ## Use PowerShell
 
-### Modify block file and URL entries in the Tenant Allow/Block List
+### Modify allow or block sender, file and URL entries in the Tenant Allow/Block List
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/modify-remove-entries-tenant-allow-block.md:80]:
```suggestion
### Modify allow or block sender, file, and URL entries in the Tenant Allow/Block List
```

File: microsoft-365/security/office-365-security/modify-remove-entries-tenant-allow-block.md [hunk 2]
```diff
@@ -91,9 +93,9 @@ Set-TenantAllowBlockListItems -ListType Url -Ids "RgAAAAAI8gSyI_NmQqzeh-HXJBywBw
 
 For detailed syntax and parameter information, see [Set-TenantAllowBlockListItems](/powershell/module/exchange/set-tenantallowblocklistitems).
 
-### Remove URL or file entries from the Tenant Allow/Block List
+### Remove allow or block sender, URL or file entries from the Tenant Allow/Block List
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/modify-remove-entries-tenant-allow-block.md:96]:
```suggestion
### Remove allow or block sender, URL, or file entries from the Tenant Allow/Block List
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dhairyya, kindly commit the minor suggested changes. Many thanks.

**Review by MSFTTracyP — APPROVED** [review]
Thanks

**Pull request merged**